blog-post-review.md

```diff
@@ -0,0 +1,261 @@
+
+## Introduction
+Since **MAGE's** latest release, you can now run **GPU** powered graph analytics from **Memgraph** in seconds. Loading the whole dataset containing 1M+ edges and getting the first graph analytics results shouldn't take more than 15 seconds :fire: On top of it all, you can do it in **Python**. Let me show you how.
+
+In this tutorial you will learn:
+* how to effortlessly import data inside Memgraph, all while using **Python**
+* how to run analytics on graphs and get results fast 
+* how to run analytics on **NVIDIA GPU** from Memgraph 
+
+What made all of it possible is the integration and collaboration of **Nvidia cuGraph** and **Memgraph** teams :heart:.
+
+In this tutorial I will show you how to use  **PageRank** graph analysis and **Louvain** community detection on a **Facebook dataset** containing 1.3 million edges. After you follow this tutorial, you will know how to run  any of the following graph analytics algorithms with **Python** on your dataset:  
+* Balanced Cut (clustering)
+* Spectral Clustering (clustering) 
+* HITS (hubs vs authorities analytics)
+* Leiden (community detection)
+* Katz Centrality 
+* Betweenness Centrality  
+
+All above algorithms are powered by **[Nvidia cuGraph](https://www.nvidia.com/en-us/)** and they will execute on **GPU**. 
+
+## Prerequisites
+
+To follow the tutorial, please install:
+- [Docker](https://docs.docker.com/get-docker/) - needed to run `mage-cugraph` image
+- [Jupyter](https://jupyter.org/install) - using jupyter-notebook you can write  `CSV` importing and graph analytics in one file
+- [GQLAlchemy](https://pypi.org/project/gqlalchemy/) - to connect to Memgraph with Python
+- [Memgraph Lab](https://memgraph.com/lab) - a GUI tool we use to to visualize graphs
+
+
+### Memgraph with Docker
+We use *[Docker](https://www.docker.com/)* because the Memgraph database is a native Linux application and cannot be installed on Windows and macOS. But for this part, we will use **Docker image** of  `mage-cugraph`. [Here](https://memgraph.com/docs/mage) you can check how to pull the image, and set everything up before running the **Memgraph-cuGraph** container.
+
+
+Before running `mage-cugraph` Docker image, you should [clone](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository) **[jupyter-memgraph-tutorials](https://github.com/memgraph/jupyter-memgraph-tutorials)** GitHub repo and position yourself inside `jupyter-memgraph-tutorials/cugraph-analytics` folder. We will do one "hack" here for later. Since we are working with *Docker* and we will read our **CSV** files, **Memgraph** needs to have access to such files. But they are currently on our machine and need to be inside *Docker*. That's we will create **Docker** volume mounting our current `data/facebook_clean_data/` folder to `/samples` folder inside the Docker container. Inside `/samples` folder, *Memgraph* will find our **CSV** files.
```

Comment:
...the jupyter-memgraph-tutorials GitHub repo...
...inside the `bla` foder.

Reply:
> If the "cloned the repo" you mention before the docker run is cloning the jupyter then follow with this
Yes this one is true 👍🏻

Reply:
> What do you mean by "read" CSV files? We'll import them, right?

Yes